pom.xml

```diff
@@ -46,6 +46,11 @@
             <artifactId>simpleclient_common</artifactId>
             <version>${prometheus-client.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_hotspot</artifactId>
+            <version>0.8.1</version>
```

Comment:
Please use the existing `prometheus-client.version` property. Bump it while you're at it.